linters/.eslintrc

```diff
@@ -0,0 +1,168 @@
+{
+  "parser": "babel-eslint",
```

Comment:
With babel-eslint I ran into issue with some stylistic checks. After babel modifies the raw code, it cannot pass some checks. That might be fixed already and worth doublechecking.